inspirehep/modules/records/serializers/pybtex_plugins/base_plugin.py

```diff
@@ -0,0 +1,153 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of INSPIRE.
+# Copyright (C) 2014-2017 CERN.
+#
+# INSPIRE is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# INSPIRE is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with INSPIRE. If not, see <http://www.gnu.org/licenses/>.
+#
+# In applying this license, CERN does not waive the privileges and immunities
+# granted to it by virtue of its status as an Intergovernmental Organization
+# or submit itself to any jurisdiction.
+
+from __future__ import absolute_import, division, print_function
+
+from pybtex.database.output import BaseWriter
+import re
+from inspirehep.utils.jinja2 import render_template_to_string_for_blueprint
+from ...views import blueprint
+from ..fields_export import eprint_new_style
+
+
+class PybtexBaseWriter(BaseWriter):
+    MAX_AUTHORS_BEFORE_ET_AL = 10  # According to CSE stylebook
```

Comment:
this is repeated in https://github.com/inspirehep/inspire-next/pull/2742/files/f1e0bbbd8530bda1d5cc244f787dcf98dac2a42a#diff-ec291a3f354a13f867e23279e47abbb7R29